src/core/client/menus/object.ts

```diff
@@ -68,7 +70,7 @@ const ObjectWheelMenuConst = {
 
         for (const callback of Injections) {
             try {
-                options = callback(hash, scriptID, options);
+                options = callback(hash, scriptID, options, item);
```

Comment:
I think this needs to be wrapped in if / else because if someone will not define item this will cause a error i am pretty sure

Reply:
Tested it with item and also without. The item is an optional parameter and will then be just null in this case.

Reply:
my test code

`export class ObjectMenu {
    static init() {
        ObjectWheelMenu.addInjection(ObjectMenu.addMenuOptions1);
        ObjectWheelMenu.addInjection(ObjectMenu.addMenuOptions2);
    }

    static addMenuOptions1(
        modelHash: number,
        scriptID: number,
        options: Array<IWheelOptionExt>,
    ): Array<IWheelOptionExt> {
        alt.logWarning('AddMenu without item');

        return options;
    }

    static addMenuOptions2(
        modelHash: number,
        scriptID: number,
        options: Array<IWheelOptionExt>,
        item?: GroundItem,
    ): Array<IWheelOptionExt> {
        if (item) alt.logWarning('AddMenu with item' + JSON.stringify(item));

        return options;
    }
}`

If items exist both callbacks will trigger. If no item is there just the first one will triggered.

Reply:
Correction: Both callbacks will be triggered, but in the second one the Item is null if there is no item